core/arch/instr_create_shared.h

```diff
@@ -49,16 +49,16 @@
 /* (deliberately not indenting the #includes in API_EXPORT_ONLY for generated file) */
 #endif
 #ifdef API_EXPORT_ONLY
-#ifdef X86
-# include "dr_ir_macros_x86.h"
-#elif defined(AARCH64)
-# include "dr_ir_macros_aarch64.h"
-#elif defined(ARM)
-# include "dr_ir_macros_arm.h"
-#endif
-#include "dr_ir_opnd.h"
-#include "dr_ir_instr.h"
-#include "dr_ir_utils.h"
+# ifdef X86
```

Comment:
Please see the comment right above about why these are deliberately not indented -- the outer define won't be there in the generated file.  There should be an exception for this.